src/traces/scattergl/attributes.js

```diff
@@ -83,9 +83,7 @@ var attrs = module.exports = overrideAll({
         marker: scatterAttrs.unselected.marker
     },
 
-    opacity: extendFlat({}, plotAttrs.opacity, {
-        editType: 'calc'
-    }),
+    opacity: extendFlat({}, plotAttrs.opacity)
```

Comment:
you don't even need the `extendFlat`, right? `overrideAll` will take care of that.

Reply:
done in https://github.com/plotly/plotly.js/pull/2442/commits/47768321e84cdbd40f4a44bda3e7cd1c0f2c2fae